Vialer/Helpers/VialerStats.swift

```diff
@@ -122,7 +122,7 @@ import Foundation
         defaultData = [
             VialerStatsConstants.APIKeys.sipUserId: SystemUser.current().sipAccount,
             VialerStatsConstants.APIKeys.os: "iOS",
-            VialerStatsConstants.APIKeys.deviceManufacturer: UIDevice.current.model,
+            VialerStatsConstants.APIKeys.deviceManufacturer: "apple",
```

Comment:
Capitalize the a please.

Reply:
Ok but in android it is also not capital: "samsung"